src/typescript/tslint.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
why added this ?

Reply:
Add tslint for better engineering system